cla_backend/apps/checker/tests/api/test_case_api.py

```diff
@@ -352,3 +359,32 @@ def test_create_should_ignore_outcome_code(self):
 
         case = Case.objects.first()
         self.assertNotEqual(case.outcome_code, "TEST")
+
+    def create_callback_no_capacity_slot(self):
+        dt = self._default_dt
+        slots = [
```

Comment:
This should be refactored so we have a constant called `NO_CAPACITY_SLOTS` which is then modified in each test. This will reduce the amount of duplicated lines.